src/syserror_use.cpp

```diff
@@ -29,6 +29,7 @@ std::string errno_to_string(int errno_copy) {
 		( (_POSIX_C_SOURCE >= 200112L) && !  _GNU_SOURCE ) \
 		||  ( defined(__MACH__) ) \
 		||  ( defined(_WIN32) || defined(__CYGWIN__) ) \
+		||  ( defined(OPEN_WRT) ) \
 	)
```

Comment:
// Flag "OPEN_WRT" above, can be set by OpenWrt build script (see doc/build-openwrt.md or similar if interested)